packages/core/nuxt-theme-module/theme/components/CartSidebar.vue

```diff
@@ -136,14 +136,14 @@ export default {
   },
   setup() {
     const { isCartSidebarOpen, toggleCartSidebar } = useUiState();
-    const { cart, removeFromCart, updateQuantity, loadCart } = useCart();
+    const { cart, removeFromCart, updateQuantity, load } = useCart();
```

Comment:
please do a renaming everywhere: `load: loadCart` instead - by doing this we'll see what exactly we are loading here, that's why we wanted to rename it to just load everywhere